Apps/Admin/Server/Services/SupportService.cs

```diff
@@ -0,0 +1,165 @@
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+// -------------------------------------------------------------------------
+namespace HealthGateway.Admin.Server.Services
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Threading.Tasks;
+    using AutoMapper;
+    using HealthGateway.Common.Constants;
+    using HealthGateway.Common.Data.Constants;
+    using HealthGateway.Common.Data.Models;
+    using HealthGateway.Common.Data.Models.ErrorHandling;
+    using HealthGateway.Common.Data.ViewModels;
+    using HealthGateway.Common.Models;
+    using HealthGateway.Common.Services;
+    using HealthGateway.Database.Constants;
+    using HealthGateway.Database.Delegates;
+    using HealthGateway.Database.Models;
+    using HealthGateway.Database.Wrapper;
+    using Microsoft.IdentityModel.Tokens;
+    using UserQueryType = HealthGateway.Admin.Common.Constants.UserQueryType;
+
+    /// <inheritdoc />
+    public class SupportService : ISupportService
+    {
+        private readonly IUserProfileDelegate userProfileDelegate;
+        private readonly IMessagingVerificationDelegate messagingVerificationDelegate;
+        private readonly IPatientService patientService;
+        private readonly IMapper autoMapper;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="SupportService"/> class.
+        /// </summary>
+        /// <param name="userProfileDelegate">The user profile delegate to interact with the DB.</param>
+        /// <param name="messagingVerificationDelegate">The Messaging verification delegate to interact with the DB.</param>
+        /// <param name="patientService">The patient service to lookup HDIDs by PHN.</param>
+        /// <param name="autoMapper">The injected automapper provider.</param>
+        public SupportService(
+            IUserProfileDelegate userProfileDelegate,
+            IMessagingVerificationDelegate messagingVerificationDelegate,
+            IPatientService patientService,
+            IMapper autoMapper)
+        {
+            this.userProfileDelegate = userProfileDelegate;
+            this.messagingVerificationDelegate = messagingVerificationDelegate;
+            this.patientService = patientService;
+            this.autoMapper = autoMapper;
+        }
+
+        /// <inheritdoc />
+        public RequestResult<IEnumerable<MessagingVerificationModel>> GetMessageVerifications(string hdid)
+        {
+            DBResult<IEnumerable<MessagingVerification>> dbResult = this.messagingVerificationDelegate.GetUserMessageVerifications(hdid);
+            IList<MessagingVerificationModel> verificationModels = this.autoMapper.Map<IList<MessagingVerificationModel>>(dbResult.Payload);
+            RequestResult<IEnumerable<MessagingVerificationModel>> result = new()
+            {
+                ResultStatus = ResultType.Error,
```

Comment:
Think this should be `ResultType.Success`.

Reply:
Addressed.